Session/Conversations/ConversationVC+Interaction.swift

```diff
@@ -150,8 +182,8 @@ extension ConversationVC:
             let call: SessionCall = viewModel.dependencies[singleton: .storage]
```

Comment:
Should be able to remove the `viewModel.dependencies[singleton: .storage].read` here since we no longer need the `db`

Reply:
Yep, good catch! Thanks!